### ISSUE TITLE
Please create a tagged release

### ISSUE THREAD

**Issue by Vaelatern** [keybase/kbfs#686]
Even a 0.1 release would make it possible for distributions that require releases to package this, and provide a wider test base.

Thank you

**Comment by mcritchlow** [comment]
Just 'bumping' this, as it were. A tagged release would be great. Thanks.

**Comment by cjb** [comment]
We've started tagging a little more often -- most recent tag is 1.0.21, last month.

**Comment by mcritchlow** [comment]
@cjb - Thanks! I see that's the case for the client repo, and that's awesome. However we'd need to package kbfs separately as a dependency. Any change kbfs will get the same treatment? 

Thanks for everything ya'll do on keybase, btw :1st_place_medal:

**Comment by cjb** [comment]
Hm, I think it'll be harder to change the KBFS flow.  Want to tell me more about what you're trying to do?

**Comment by Vaelatern** [comment]
I'm trying to package kbfs, because the client code as built does not come with kbfs.

**Comment by Vaelatern** [comment]
Just a single tagged commit would be enough for this.

**Comment by Vaelatern** [comment]
@cjb 
Most distros, when packaging, do not take multiple code-bases and turn them into a single tool. Most often it's the single code-base that's split up into multiple "flavors" "subpackages" or what-have-you. When multiple code-bases are important relative to each other, packagers often like to build them separately and use the mechanisms afforded by the package manager to handle dependencies.

Without versions tagged, it becomes very hard to package kbfs for anybody who does not download binary blobs from your website. The tags in this repo don't even have to match what the client currently has, or will ever have. A tag of `0.0.1` would be better than no tag, since at least the tag is a single point in history.

Why do we need tagging at all? Invariably, contributors say "Look, there was a new commit, please update this!" and the whole package has to be vetted or at minimum, blindly approved again. Version tags, and a policy of enforcing them, are a way to rate limit requests for updates, as well as a way to say "We did not choose this arbitrarily. Wait for the next release for the next feature."

There are other benefits to downstream packagers. For example, there are tools that look for the latest version of a piece of software. Reworking those tools to be aware of git commits is a lot of work or against policy, so updates to untagged software don't happen. Tagging a new release is a harmless way for upstream to signal any downstream packagers that there is new code to be reviewed and packaged appropiately.

In short: Please push a tag to the repository so we may package this software for our users.

**Comment by Vaelatern** [comment]
Two months later, is it possible to pick an arbitrary point and just tag it please?

**Comment by strib** [comment]
Ok I just made a tag corresponding to our latest desktop release: https://github.com/keybase/kbfs/releases/tag/v1.0.34

**Comment by Vaelatern** [comment]
Thank you so much!
